src/redux/Categories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      <h1>Under Construction!</h1>
+    </div>
+  );
+}
+
+export default Categories;
```

Comment:
- [ ] Our components should live in the components folder, not the redux folder. Kindly move this **`Categories`** component to the components directory.